### PR TITLE
Add ID localization for task/dns-debugging-resolution

### PR DESCRIPTION
**Pull request by danninov**
Add ID localization for dns-debugging-resolution. Issue #22296

### CODE REVIEW

File: content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+2018/08/15 14:37:15 [INFO] CoreDNS-1.2.0
+2018/08/15 14:37:15 [INFO] linux/amd64, go1.10.3, 2e322f6
+CoreDNS-1.2.0
+linux/amd64, go1.10.3, 2e322f6
+2018/09/07 15:29:04 [INFO] plugin/reload: Running configuration MD5 = 162475cdf272d8aa601e6fe67a6ad42f
+2018/09/07 15:29:04 [INFO] Reloading complete
+172.17.0.18:41675 - [07/Sep/2018:15:29:11 +0000] 59925 "A IN kubernetes.default.svc.cluster.local. udp 54 false 512" NOERROR qr,aa,rd,ra 106 0.000066649s
+```
+
+## Masalah Dikenal
```
Comment (ariscahyadi) [inline content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md:237]:
```suggestion
## Isu-isu yang Dikenal
```

File: content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+Beberapa distribusi Linux (contoh Ubuntu) menggunakan _resolver_ DNS lokal secara _default_ (systemd-resolved).
+Systemd-resolved memindahkan dan mengganti `/etc/resolv.conf` dengan berkas _stub_ yang dapat menyebabkan _forwarding loop_ yang fatal saat meresolusi nama pada server _upstream_. Ini dapat diatasi secara manual dengan menggunakan _flag_ kubelet `--resolv-conf`
+untuk mengarahkan ke `resolv.conf`  yang benar (Pada `systemd-resolved`, ini berada di `/run/systemd/resolve/resolv.conf`).
+kubeadm akan otomatis mendeteksi `systemd-resolved`, dan menyesuaikan _flag_ kubelet sebagai mana mestinya.
+
+Pemasangan Kubernetes tidak menggunakan berkas `resolv.conf` pada _node_ untuk digunakan sebagai klaster DNS secara _default_, karena proses ini umumnya spesifik pada distribusi tertentu. Hal ini bisa jadi akan diimplementasi nantinya.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md:244]:
```suggestion
Pemasangan Kubernetes tidak menggunakan berkas `resolv.conf` pada _node_ untuk digunakan sebagai klaster DNS secara bawaan, karena proses ini umumnya spesifik pada distribusi tertentu. Hal ini bisa jadi akan diimplementasi nantinya.
```

File: content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+
+Libc Linux (alias glibc) secara _default_ memiliki batasan `nameserver` DNS sebanyak 3 rekaman (_records_). Selain itu, pada glibc versi yang lebih lama dari glibc-2.17-222 ([versi terbaru lihat isu ini](https://access.redhat.com/solutions/58028)), jumlah rekaman DNS `search` dibatasi sejumlah 6 ([lihat masalah sejak 2005 ini](https://bugzilla.redhat.com/show_bug.cgi?id=168253)). Kubernetes membutuhkan 1 `nameserver` rekaman dan 3 `search` rekaman. Ini berarti jika instalasi lokal telah menggunakan 3 `nameserver` atau menggunakan lebih dari 3 `search`,sementara versi glibc kamu termasuk yang terkena dampak, beberapa dari pengaturan tersebut akan hilang. Untuk menyiasati batasan rekaman DNS `nameserver`, _node_ dapat menjalankan `dnsmasq`,yang akan menyediakan `nameserver` lebih banyak. Kamu juga dapat menggunakan kubelet `--resolv-conf` _flag_. Untuk menyiasati batasan rekaman `search`, pertimbangkan untuk memperbarui distribusi linux kamu atau memperbarui glibc ke versi yang tidak terdampak.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md:246]:
```suggestion
Libc Linux (alias glibc) secara bawaan memiliki batasan `nameserver` DNS sebanyak 3 rekaman (_records_). Selain itu, pada glibc versi yang lebih lama dari glibc-2.17-222 ([versi terbaru lihat isu ini](https://access.redhat.com/solutions/58028)), jumlah rekaman DNS `search` dibatasi sejumlah 6 ([lihat masalah sejak 2005 ini](https://bugzilla.redhat.com/show_bug.cgi?id=168253)). Kubernetes membutuhkan 1 `nameserver` rekaman dan 3 `search` rekaman. Ini berarti jika instalasi lokal telah menggunakan 3 `nameserver` atau menggunakan lebih dari 3 `search`,sementara versi glibc kamu termasuk yang terkena dampak, beberapa dari pengaturan tersebut akan hilang. Untuk menyiasati batasan rekaman DNS `nameserver`, _node_ dapat menjalankan `dnsmasq`,yang akan menyediakan `nameserver` lebih banyak. Kamu juga dapat menggunakan kubelet `--resolv-conf` _flag_. Untuk menyiasati batasan rekaman `search`, pertimbangkan untuk memperbarui distribusi linux kamu atau memperbarui glibc ke versi yang tidak terdampak.
```

File: content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+Jika kamu menggunakan Alpine versi 3.3 atau lebih lama sebagai dasar _image_ kamu, DNS mungkin tidak dapat bekerja dengan benar disebabkan masalah dengan Alpine.
+[Masalah 30215](https://github.com/kubernetes/kubernetes/issues/30215) Kubernetes menyediakan informasi lebih detil tentang ini.
+
+## {{% heading "whatsnext" %}}
+
+- Lihat [Autoscaling the DNS Service in a Cluster](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/).
```
Comment (ariscahyadi) [inline content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md:253]:
```suggestion
- Lihat [Penyekalaan otomatis Service DNS dalam klaster](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/).
```

File: content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+- Baca [DNS for Services and Pods](/docs/concepts/services-networking/dns-pod-service/)
```
Comment (ariscahyadi) [inline content/id/docs/tasks/administer-cluster/dns-debugging-resolution.md:254]:
```suggestion
- Baca [DNS untuk Service dan Pod](/id/docs/concepts/services-networking/dns-pod-service/)
```

**Comment by irvifa** [comment]
Thanks 🎉 
/approve

**Pull request merged**